CIPs/cip-0032.md

```diff
@@ -43,23 +44,22 @@ The `Attestations` contract will be upgraded to register the request and complet
 ```js
 // Tracks per issuer completion rate
 // These values are reset per issuer from Validators.distributeEpochPaymentsFromSigner()
-mapping(address => uint256) public attestationRequestCount;
-mapping(address => uint256) public attestationCompletionCount;
+mapping(address => uint64) public attestationRequestCount;
+mapping(address => uint64) public attestationCompletionCount;
```

Comment:
Because [mappings store their values at sparse memory locations](https://docs.soliditylang.org/en/v0.4.24/miscellaneous.html), the value storage cannot benefit from packing. As a result, using `uint64` as the mapping value will use the same about of storage and slightly more computation to access it.

Reply:
Oh thanks, good to know. The quirks of Solidity...